motr/st/utils/helper.c

```diff
@@ -219,19 +218,16 @@ static int open_entity(struct m0_entity *entity)
 
 	rc = m0_entity_open(entity, &ops[0]);
 	if (rc != 0)
-		goto cleanup;
+		return M0_ERR(rc);
 
 	m0_op_launch(ops, 1);
 	rc = m0_op_wait(ops[0], M0_BITS(M0_OS_FAILED,
-					       M0_OS_STABLE),
-			       M0_TIME_NEVER);
+					M0_OS_STABLE), M0_TIME_NEVER);
 	if (rc == 0)
-		rc = ops[0]->op_rc;
+		rc = m0_rc(ops[0]);
```

Comment:
Personally, I am thinking the function name should be changed to m0_op_rc().

Reply:
I agree. It can be done in another PR.